src/it/resources/org/checkstyle/suppressionxpathfilter/missingswitchdefault/SuppressionXpathRegressionMissingSwitchDefaultTwo.java

```diff
@@ -0,0 +1,18 @@
+package org.checkstyle.suppressionxpathfilter.missingswitchdefault;
+
+public class SuppressionXpathRegressionMissingSwitchDefaultTwo {
```

Comment:
Please also update unit tests with example of nested switch

Reply:
Should I nest another switch inside another Switch's case?

Reply:
I had nested a switch inside a switch default.

Reply:
It can be both.
In report I saw examples of nested switch, so it is good to have them in tests

Reply:
@Abhishek-kumar09 I was saying about unit tests, not xpath regression tests. Please correct